featurebyte/service/feature_store_warehouse.py

```diff
@@ -202,4 +202,7 @@ async def list_columns(
         except db_session.no_schema_error as exc:
             raise TableNotFoundError(f"Table {table_name} not found.") from exc
 
-        return [ColumnSpec(name=name, dtype=dtype) for name, dtype in table_schema.items()]
+        return [
+            ColumnSpec(name=name, dtype=schema.dtype, description=schema.description)
+            for name, schema in table_schema.items()
+        ]
```

Comment:
We could add a new schema like `ColumnSpecWithDescription` at `featurebyte/query_graph/model/column_info.py`, and then use the schema here without modifying existing `ColumnSpec` schema. By doing so, we could avoid modifying underlying query graph (should not see lots of changes due to changes in hash values).